tests/foreman/ui/test_rhc.py

```diff
@@ -0,0 +1,193 @@
+"""Test class for RH Cloud connector - rhc
+
+:Requirement: Remoteexecution
+
+:CaseAutomation: Automated
+
+:CaseLevel: Acceptance
+
+:CaseComponent: RHCloud-CloudConnector
+
+:Assignee: jpathan
+
+:TestType: Functional
+
+:CaseImportance: High
+
+:Upstream: No
+"""
+import pytest
+from wait_for import wait_for
+
+from robottelo import constants
+from robottelo.config import settings
+from robottelo.constants import DEFAULT_SUBSCRIPTION_NAME
+from robottelo.datafactory import gen_string
+from robottelo.helpers import file_downloader
+from robottelo.logging import logger
+
+
+@pytest.fixture(scope='module')
+def subscribe_rhc_satellite(clean_rhsm, module_target_sat):
+    """subscribe rhc satellite to cdn"""
+    if module_target_sat.os_version.major < 8:
+        release_version = f'{module_target_sat.os_version.major}Server'
+    else:
+        release_version = f'{module_target_sat.os_version.major}'
+    module_target_sat.register_contenthost(
+        org=None,
+        lce=None,
+        username=settings.subscription.rhn_username,
+        password=settings.subscription.rhn_password,
+        releasever=release_version,
+    )
+    result = module_target_sat.subscription_manager_attach_pool([settings.subscription.rhn_poolid])[0]
+    if 'Successfully attached a subscription' in result.stdout:
+        # extras is not in RHEL8: https://access.redhat.com/solutions/5331391
+        if module_target_sat.os_version.major < 8:
+            module_target_sat.enable_repo(
+                f'rhel-{module_target_sat.os_version.major}-server-extras-rpms', force=True
+            )
+        yield
+    else:
+        pytest.fail('Failed to attach system to pool. Aborting Test!.')
+
+
+@pytest.fixture()
```

Comment:
We can move it to `subscribe_satellite` and `module_subscribe_satellite` fixtures ([subscription.py#L16](https://github.com/SatelliteQE/robottelo/blob/master/pytest_fixtures/component/subscription.py#L16)) but it won't help in case of this test scenario. Those fixtures contain code for unregistering Satelite, which we don't want to do. 
If you want I can add it there but we won't be using them in this test.

Reply:
Agree ! And moving this fixture to global can be handled outside of this PR !